server/src/main/java/org/elasticsearch/cluster/coordination/UnsafeBootstrapMasterCommand.java

```diff
@@ -0,0 +1,187 @@
+import org.elasticsearch.common.settings.Setting;
+import org.elasticsearch.common.settings.Settings;
+import org.elasticsearch.common.xcontent.NamedXContentRegistry;
+import org.elasticsearch.env.Environment;
+import org.elasticsearch.env.NodeEnvironment;
+import org.elasticsearch.env.NodeMetaData;
+import org.elasticsearch.node.Node;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.Objects;
+
+public class UnsafeBootstrapMasterCommand extends EnvironmentAwareCommand {
+
+    private final NamedXContentRegistry namedXContentRegistry;
+
+    static final String WARNING_MSG =
+            "-----------------------------------------------------------------------\n" +
+            "\n" +
+            "    WARNING: Elasticsearch MUST be stopped before running this tool.\n" +
+            "\n" +
+            "You should run this tool only if you've lost the majority of master eligible nodes.\n" +
+            "If you have a backup, restore from the backup instead.\n" +
+            "Running this tool should be the last resort.\n" +
+            "Running this tool may cause arbitrary data loss and may render your cluster completely non functional.\n" +
+            "Do you accept this risk?\n";
+    static final String ABORTED_BY_USER_MSG = "aborted by user";
+    static final String NOT_MASTER_NODE_MSG = "unsafe-bootstrap tool can only be run on master eligible node";
+    static final String FAILED_TO_OBTAIN_NODE_LOCK_MSG = "failed to lock node's directory, is Elasticsearch still running?";
+    static final String NO_NODE_FOLDER_FOUND_MSG = "no node folder is found in data folder(s), node has not been started yet?";
+    static final String NO_NODE_METADATA_FOUND_MSG = "no node meta data is found, node has not been started yet?";
+    static final String NO_MANIFEST_FILE_FOUND_MSG = "no manifest file is found, do you run pre 7.0 Elasticsearch?";
+    static final String GLOBAL_GENERATION_MISSING_MSG = "no metadata is referenced from the manifest file, cluster has never been " +
+            "bootstrapped?";
+    static final String NO_GLOBAL_METADATA_MSG = "failed to find global metadata, metadata corrupted?";
+    static final String EMPTY_LAST_COMMITTED_VOTING_CONFIG_MSG =
+            "last committed voting voting configuration is empty, cluster has never been bootstrapped?";
+    static final String WRITE_METADATA_EXCEPTION_MSG = "exception occurred when writing new metadata to disk";
+    static final String MASTER_NODE_BOOTSTRAPPED_MSG = "Master node was successfully bootstrapped";
+    static final Setting<String> UNSAFE_BOOTSTRAP =
+            ClusterService.USER_DEFINED_META_DATA.getConcreteSetting("cluster.metadata.unsafe-bootstrap");
+
+    UnsafeBootstrapMasterCommand() {
+        super("Unsafely bootstraps the master node if the majority of master eligible nodes is lost");
```

Comment:
Again, not strictly a majority. Maybe we should replace "bootstraps" with "forces the successful election of"?

Reply:
How do you find this message https://github.com/elastic/elasticsearch/pull/37696/commits/b6ada540cf2d2245bbff85c8d4553b5acfb28011?